packages/blade/src/components/Chip/AnimatedChip.native.tsx

```diff
@@ -22,8 +22,8 @@ const AnimatedChip = ({
 }: Omit<AnimatedChipProps, 'theme'>): React.ReactElement => {
   const { theme } = useTheme();
 
-  const easing = getIn(theme, chipMotionTokens.easing);
-  const duration = getIn(theme, chipMotionTokens.duration);
+  const easing = getIn(theme.motion, chipMotionTokens.easing);
```

Comment:
was this broken earlier?

Reply:
No, just refactored types as per https://github.com/razorpay/blade/pull/1948#discussion_r1449902110